service-base/src/main/java/io/narayana/lra/logging/LraI18nLogger.java

```diff
@@ -135,7 +135,7 @@ String info_failedToEnlistingLRANotFound(URL lraId, URI coordinatorUri, int coor
     @Message(id=25031, value = "%s: Invalid link URI (%s): missing compensator or after LRA callback")
     String error_missingCompensator(URI id, String linkURI);
 
-    @Message(id = 25032, value = "LRA Record: Cannot save state (reason: %s)")
+    @Message(id = 25032, value = "LRA Record: Cannot save state, %s")
```

Comment:
I don't normally like to change existing log messages, is removing the bracketed reason strictly necessary?

Reply:
No it's not - I'll revert it.